package.json

```diff
@@ -32,7 +32,7 @@
     "envelope-spec": "^1.1.0",
     "lodash.get": "^4.4.2",
     "lodash.set": "^4.3.2",
-    "private-group-spec": "^1.1.3",
+    "private-group-spec": "powersource/private-group-spec#spec-tests",
```

Comment:
TODO

Reply:
@mixmix @staltz need a new minor version of private-group-spec

Reply:
@Powersource Done, v1.2.0